### PR TITLE
[5.3] Sugggest allowing an $options array to be passed through the Eloquent model's create method

### PR DESCRIPTION
**Pull request by chadanuk**

### CODE REVIEW

**Comment by akiyamaSM** [comment]
can I know for what?

File: tests/Database/DatabaseEloquentIntegrationTest.php [hunk 1]
```diff
@@ -377,7 +377,7 @@ public function testFindOrFailWithMultipleIdsThrowsModelNotFoundException()
     {
         EloquentTestUser::create(['id' => 1, 'email' => 'taylorotwell@gmail.com']);
         EloquentTestUser::findOrFail([1, 2]);
-    }
+    }
```
Comment (GrahamCampbell) [inline tests/Database/DatabaseEloquentIntegrationTest.php]: oops

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Database/DatabaseEloquentIntegrationTest.php (hunk 1 above)
Comment (chadanuk) [inline tests/Database/DatabaseEloquentIntegrationTest.php]: Thanks! I'd been looking for a test of the save method and passing through $options in particular and must have tweaked the file inadvertantly!

**Comment by chadanuk** [comment]
My thinking is partly down to some in-app logic in terms of using the $options variable in an overwritten version of the save method in one of my models. I'm then creating an instance at another point with create, and it would be convenient to pass options through here as well.

The more general thought on this is regarding consistency; if there is an $options variable in the save method then it makes sense to be able to pass these through to save as an expected behaviour.

**Comment by taylorotwell** [comment]
This change can't really be made on 5.3 since it is a breaking change.

**Pull request closed without merging**